tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -271,69 +271,72 @@ protected override void OnNext(string eventName, object arg)
 #endif
 
         private static string SimplifyRoutePattern(
-            RoutePattern routePattern,
+            RoutePattern? routePattern,
```

Comment:
I _think_ we can simplify this quite a lot. If `routePattern` is null, then that's because we haven't matched an endpoint. If we haven't matched an endpoint, then we can't extract route parameters, and so it doesn't make sense to call this method. I think we can revert the changes in this one, and bypass it when `RoutePattern` is `null`?

Reply:
Done in 9e02c49 thank you